packages/kestrel_core/src/kestrel/mapping/data_model.py

```diff
@@ -364,6 +389,29 @@ def translate_attributes_projection_to_ocsf(
     return tuple(final_result)
 
 
+def _get_transformer(obj: dict) -> Optional[Union[str, dict]]:
+    # First look for native -> ocsf value transformer
+    ocsf_value = obj.get("ocsf_value")
+    if not ocsf_value:
+        # Next check if there's a ocsf -> native transformer
+        native_value = obj.get("native_value")
+        if isinstance(native_value, dict):
+            # If it's a simple value map, we can reverse it
+            ocsf_value = _reverse_value_map(native_value)
+    return ocsf_value
+
+
+def _get_transformers(mappings: List[Union[str, dict]]) -> List[Union[str, dict]]:
+    """Collect value transformers from mappings, discarding duplicates"""
+    result = []  # Need a list instead of set since dicts aren't hashable
+    for mapping in mappings:
+        if "ocsf_value" in mapping or "native_value" in mapping:
+            transformer = _get_transformer(mapping)
+            if transformer not in result:
```

Comment:
Possible for `_get_transformer()` to return `None`? e.g. https://github.com/opencybersecurityalliance/kestrel-lang/blob/develop/packages/kestrel_core/src/kestrel/mapping/fields/ecs.yaml#L88

May not append `None` to `result`.

Reply:
Good catch